src/components/ItemList/ItemRow.tsx

```diff
@@ -1,21 +1,30 @@
 import React from 'react';
-import DatalistItem from 'box-ui-elements/es/components/datalist-item';
+import classNames from 'classnames';
+import omit from 'lodash/omit';
 import { UserMini, GroupMini } from '../../@types';
 import './ItemRow.scss';
 
 export type Props = {
-    id?: string;
+    className?: string;
+    isActive?: boolean;
     item?: UserMini | GroupMini;
-    name?: string;
 };
 
-const ItemRow = ({ item, ...rest }: Props): JSX.Element | null => {
+const ItemRow = ({ className, isActive, item, ...rest }: Props): JSX.Element | null => {
     if (!item || !item.name) {
         return null;
     }
 
+    const itemProps = omit(rest, Object.keys(item));
```

Comment:
This is pretty confusing. Can we either pass `item` in as a discrete prop _or_ as a spread set of props, rather than both?